config/pipeline.yaml

```
@@ -552,3 +653,21 @@ build_configs:
     tree: mediatek
     branch: 'for-next'
     variants: *build-variants
+
+  android_mainline:
+    tree: android
+    branch: 'android-mainline'
+    build_environment: gcc-10
+    arch: armel
+    defconfig: multi_v7_defconfig
+    filters:
+      - regex: { defconfig: 'multi_v7_defconfig' }
```

```suggestion
    variants: *build-variants
```

No need to define additional parameters here, this section is only used to define tree/branch combinations we want to monitor. The important bit (defconfig) should be defined in the `kbuild` job.